frontend/src/components/Home.js

```diff
@@ -1,45 +1,61 @@
-import React from "react";
+import React, { Component } from "react";
 import { Layout, BackTop, Button, Icon } from "antd";
 import LoadingBar from "react-redux-loading-bar";
 import DashboardRoutes from "@/routes/DashboardRoutes";
 import AdminDashboardRoutes from "@/routes/AdminDashboardRoutes";
 import { AuthenticationGuard } from "@/HOC/AuthenticationGuard";
 import NavBar from "./navigation/NavBar";
+import WarningBanner from "@/components/common/WarningBanner";
+import { detectIE, detectTestEnvironment } from "@/utils/environmentUtils";
 
 /**
  * @class Home contains the navigation and wraps the Dashboard routes. Home should not contain any redux logic/state.
  * Home is wrapped in AuthenticationGuard which checks keycloak authorization.
  */
 
-export const Home = () => {
-  const { Content } = Layout;
-  return (
-    <Layout className="layout">
-      <div className="header">
-        <NavBar />
-        <LoadingBar
-          style={{
-            backgroundColor: "#6b6363",
-            position: "fixed",
-            top: 55,
-            zIndex: 90,
-            width: "100%",
-            height: "8px",
-          }}
-        />
-      </div>
-      <Content className="content">
-        <DashboardRoutes />
-        <AdminDashboardRoutes />
-        <BackTop>
-          <Button type="primary">
-            <Icon type="arrow-up" theme="outlined" />
-          </Button>
-        </BackTop>
-      </Content>
-      <div className="footer" />
-    </Layout>
-  );
-};
+export class Home extends Component {
+  state = { isIE: false, isTest: false };
+
+  componentDidMount() {
+    this.setState({ isIE: detectIE(), isTest: detectTestEnvironment() });
+  }
+
+  handleIEClose = () => {
+    // this.setState({ isIE: false });
```

Comment:
This looks like it should be removed or enabled

Reply:
uncommented!